pkg/cli/options_test.go

```diff
@@ -40,7 +40,7 @@ var _ = Describe("CLI options", func() {
 
 		p   = newMockPlugin(pluginName, pluginVersion, projectVersion)
 		np1 = newMockPlugin("Plugin", pluginVersion, projectVersion)
-		np2 = mockPlugin{pluginName, plugin.Version{Number: -1, Stage: plugin.StableStage}, []string{projectVersion}}
```

Comment:
why the plugin stage was removed?

Reply:
Stable is the zero-value for stage so it is implicitly the same and that way we avoid an extra import